router/image/templates/nginx.conf

```diff
@@ -158,6 +158,10 @@ http {
     {{ $useSSL := or (getv "/deis/router/sslCert") "false" }}
     {{ $domains := ls "/deis/domains" }}
     {{ $certs := ls "/deis/certs" }}
+    {{ if exists "/deis/router/affinityArg" }}
+    set_random $prng 0 99;
+    set_if_empty $arg_{{ $affinityArg }} $prng;
```

Comment:
I'm missing where `$affinityArg` gets initialized. Should this be `getv "/deis/router/affinityArg"` instead?